ietf/group/management/commands/import_iesg_statements.py

```diff
@@ -93,20 +97,34 @@ def handle(self, *args, **options):
                 os.makedirs(dest.parent, exist_ok=True)
                 shutil.copy(source, dest)
 
+            redirects.append(
+                [
+                    f"www.ietf.org/about/groups/iesg/statements/{item.slug}",
+                    f"https://datatracker.ietf.org/group/iesg/statements/{name}",
+                    302,
+                ]
+            )
+
         shutil.rmtree(tmpdir)
+        with open("iesg_statement_redirects.csv", "w", newline="") as f:
+            csvwriter = csv.writer(f)
+            for row in redirects:
+                csvwriter.writerow(row)
 
     def get_work_items(self):
-        Item = namedtuple("Item", "doc_time source_filename title")
+        Item = namedtuple("Item", "doc_time source_filename title slug")
         items = []
         dressed_rows = " ".join(
             self.cut_paste_from_www().expandtabs(1).split(" ")
         ).split("\n")
+        old_slugs = self.get_old_slugs()
         # Rube-Goldberg-esque dance to deal with conflicting directions of the scrape and
         # what order we want the result to sort to
         dressed_rows.reverse()
+        old_slugs.reverse()
         total_times_date_seen = Counter([row.split(" ")[0] for row in dressed_rows])
         count_date_seen_so_far = Counter()
-        for row in dressed_rows:
+        for row, slug in zip(dressed_rows, old_slugs):
```

Comment:
Might want to validate that `len(dressed_rows) == len(old_slugs)` because `zip()` will happily cut off at the shorter of the lists

Reply:
the input is static and I've verified the lengths match by hand.